### ISSUE TITLE
A probable issue with Interactive Latent (Upscale) node.

### ISSUE THREAD

**Issue by mart-hill** [comfyanonymous/ComfyUI#2226]
I've noticed a weird UI behavior after todays updates - the UI freezes after 'reaching' the mentioned node, which never happened before:
![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/e44c88b2-72c3-4cca-ab91-cc74e1ee1c2e)
[woodland.json](https://github.com/comfyanonymous/ComfyUI/files/13620971/woodland.json)

Before I execute it, the workflow looks like this:
![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/a09dd873-5ad9-4568-a5f6-46be1520cfdd)

![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/46d783ef-ea5a-4364-939d-618442fa630f)

The only thing I can move is the main menu bar, everything else is frozen, until I refresh the page.

F12 shows:
```
Uncaught TypeError: ctx.save is not a function
    at nodeType.onDrawForeground (impact-pack.js:65:8)
    at drawBadge (comfyui-manager.js:283:25)
    at node.onDrawForeground (comfyui-manager.js:955:45)
    at LGraphCanvas.drawNode (litegraph.core.js:8518:18)
    at LGraphCanvas.drawNode (snapToGrid.js:86:24)
    at LGraphCanvas.drawNode (app.js:1146:29)
    at LGraphCanvas.drawNode (snapToGrid.js:32:21)
    at LGraphCanvas.drawFrontCanvas (litegraph.core.js:7801:22)
    at LGraphCanvas.draw (litegraph.core.js:7716:18)
    at LGraphCanvas.renderFrame (litegraph.core.js:5783:22)
toolbarContentscript.js:113 {"message":"A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received"}
:8188/view?filename=ComfyUI_00097.webp&type=output&subfolder=c%3A%5CComfyUI%5CComfyUI%5CComfyUI%5Coutput%5Ctxt2img-images%5C2023-12-08&t=1702069303380:1 
        
        
       Failed to load resource: the server responded with a status of 403 (Forbidden)
logging.js:315 ObjectanotherQueueWaiting: falseautoQueueInProgress: false[[Prototype]]: Object
view:1 
        
        
       Failed to load resource: the server responded with a status of 403 (Forbidden)
```
I've redirected the image save to symlinked NAS folder, but that's about it - the images displayed correctly all this time. 🙂 
Despite this issue, the image generation finishes successfully, and there are no errors in Powershell window.

ComfyUI: 1795 [a4ec54] (2023-12-08)
Manager: V1.10

**Comment by gmorks** [comment]
I'm having a similar problem, about the freezing, but I'm not using the latent upscale and my F12 gives me another error. I'm curious about the freezing it happens when I zoom out.

```
Uncaught TypeError: canvas.ds is undefined
    onDrawForeground http://127.0.0.1:8188/extensions/rgthree-comfy/reroute.js:80
    drawBadge http://127.0.0.1:8188/extensions/ComfyUI-Manager/comfyui-manager.js:283
    onDrawForeground http://127.0.0.1:8188/extensions/ComfyUI-Manager/comfyui-manager.js:963
    drawNode http://127.0.0.1:8188/lib/litegraph.core.js:8518
    drawNode http://127.0.0.1:8188/extensions/core/snapToGrid.js:86
    drawNode http://127.0.0.1:8188/scripts/app.js:1146
    drawNode http://127.0.0.1:8188/extensions/cg-use-everywhere/use_everywhere.js:233
    drawNode http://127.0.0.1:8188/extensions/ComfyUI-Custom-Scripts/js/snapToGrid.js:32
    drawFrontCanvas http://127.0.0.1:8188/lib/litegraph.core.js:7801
    draw http://127.0.0.1:8188/lib/litegraph.core.js:7716
    draw Choo
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5783
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
    renderFrame http://127.0.0.1:8188/lib/litegraph.core.js:5788
reroute.js:80:33

```

[workflow_freezing.json](https://github.com/comfyanonymous/ComfyUI/files/13622055/workflow_freezing.json)

**Comment by mart-hill** [comment]
I reported this bug to the Comfy-UI-Impact dev, too, and he [responded](https://github.com/ltdrdata/ComfyUI-Impact-Pack/issues/349#issuecomment-1847996440), that he needs to fix this bug. It's possible, that there's something more to it, though.🙂

**Comment by ltdrdata** [comment]
This is fixed on ComfyUI-Manager V1.10.3.
Please update ComfyUI-Managr.

**Comment by mart-hill** [comment]
Thank you!
